docs/resources/aviatrix_account.md

```diff
@@ -23,7 +23,7 @@ resource "aviatrix_account" "temp_acc_aws" {
   aws_iam            = true
   aws_role_app       = "arn:aws:iam::123456789012:role/aviatrix-role-app"
   aws_role_ec2       = "arn:aws:iam::123456789012:role/aviatrix-role-ec2"
-  group_names        = ["write_only"]
+  rbac_groups        = ["write_only"]
```

Comment:
By adding the `rbac_groups` attribute to every example in the documentation, we are effectively saying that by default the `write_only` RBAC group should be used. Is this the intended default group or should we just add a separate example section for setting RBAC groups? What do you guys think @shuoliugit @zhiqiangjin @inc1t3Ful ?